docs/repl.md

```diff
@@ -15,6 +15,12 @@ This would open a Chrome browser that you can control with the REPL interface. M
 wdio repl chrome -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY
 ```
 
+```sh
+wdio repl appium --config './path/appium.conf.js'
```

Comment:
Can you please add an example of appium.conf.js ?

Reply:
anybody have a working example maybe ? @mgrybyk